tests/python/unittest/test_operator.py

```diff
@@ -1915,7 +1915,11 @@ def gen_broadcast_data(idx):
         [[1, 1, 65, 2, 22], [1, 1, 65, 1, 1]],
         [[1, 24, 103, 17, 18], [1, 24, 1, 1, 1]],
         [[1, 1, 1, 1, 2], [1, 24, 194, 50, 1]],
-        [[1, 1, 107, 84, 9], [1, 1, 1, 1, 1]]])
+        [[1, 1, 107, 84, 9], [1, 1, 1, 1, 1]],
+        [[8, 1, 6, 1], [7, 1, 5]], [[5, 4], [1]],
+        [[256, 256, 3], [3]], [[5, 4], [4]],
+        [[15, 3, 5], [3, 5]], [[15, 3, 5], [1, 5]],
+        [[15, 3, 5], [3, 1]]])
```

Comment:
maybe some shapes with zeros?

Reply:
These are mx.nd operator tests and this module doesn't support zero shapes.

Reply:
tests/python/unittest/test_numpy_op.py::test_np_binary_funcs

Reply:
please check if it works when rhs shape is longer than lhs e.g. [15,3] and [4, 15, 3]